### PR TITLE
chore(jest): Upgrade jest to 25.1.0

### PR DESCRIPTION
**Pull request by j3tan**
Basic Perf improvements locally

Jest 24
```
Cold: 126s
Warm: 96s
```

Jest 25
```
Cold: 104s
Warm: 66s
```

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @j3tan has signed the CLA. Thanks for the pull request!

File: package.json [hunk 1]
```diff
@@ -220,7 +220,7 @@
         "immutable": "^3.7.4",
         "intl": "^1.2.5",
         "intl-messageformat": "^2.2.0",
-        "jest": "^24.9.0",
+        "jest": "25.1.0",
```
Comment (j3tan) [inline package.json]: gonna use ^

**Comment by j3tan** [comment]
note: `jest-image-plugin` has a peerDep on `jest<=24` which is annoying. It seems to be benign

**Comment by priyajeet** [comment]
Created PR for them https://github.com/americanexpress/jest-image-snapshot/pull/170

**Pull request merged**